front/src/types/ILocation.ts

```diff
@@ -0,0 +1,4 @@
+export default interface ILocation {
+  longitude: string;
+  latitude: string;
```

Comment:
Is this normal ? Shouldn't it be a `number` ?

Reply:
Mhmm me bête